modules/rosa-sts-creating-a-cluster-with-customizations-ocm.adoc

```diff
@@ -80,20 +78,22 @@ I: Creating ocm role
 ? Role prefix: ManagedOpenShift <1>
 ? Enable admin capabilities for the OCM role (optional): No <2>
 ? Permissions boundary ARN (optional):  <3>
-? Role creation mode: auto <4>
+? Role Path (optional): <4>
+? Role creation mode: auto <5>
 I: Creating role using 'arn:aws:iam::<aws_account_id>:user/<aws_username>'
 ? Create the 'ManagedOpenShift-OCM-Role-<red_hat_organization_external_id>' role? Yes
 I: Created role 'ManagedOpenShift-OCM-Role-<red_hat_organization_external_id>' with ARN 'arn:aws:iam::<aws_account_id>:role/ManagedOpenShift-OCM-Role-<red_hat_organization_external_id>'
 I: Linking OCM role
 ? OCM Role ARN: arn:aws:iam::<aws_account_id>:role/ManagedOpenShift-OCM-Role-<red_hat_organization_external_id>
-? Link the 'arn:aws:iam::<aws_account_id>:role/ManagedOpenShift-OCM-Role-<red_hat_organization_external_id>' role with organization '<red_hat_organization_id>'? Yes <5>
+? Link the 'arn:aws:iam::<aws_account_id>:role/ManagedOpenShift-OCM-Role-<red_hat_organization_external_id>' role with organization '<red_hat_organization_id>'? Yes <6>
 I: Successfully linked role-arn 'arn:aws:iam::<aws_account_id>:role/ManagedOpenShift-OCM-Role-<red_hat_organization_external_id>' with organization account '<red_hat_organization_id>'
 ----
-<1> Specifies the prefix to include in the {cluster-manager} IAM role name. The default is `ManagedOpenShift`.
-<2> Enables the admin {cluster-manager} IAM role, which is equivalent to specifying the `--admin` argument. The admin role is required if you want to use *Auto* mode to automatically provision the cluster-specific Operator roles and the OIDC provider by using {cluster-manager}.
-<3> Optional: Specifies a permissions boundary Amazon Resource Name (ARN) for the role. For more information, see link:https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_boundaries.html[Permissions boundaries for IAM entities] in the AWS documentation.
-<4> Selects the role creation mode. You can use `auto` mode to automatically create the {cluster-manager} IAM role and link it to your Red Hat organization account. In `manual` mode, the `rosa` CLI generates the `aws` commands needed to create and link the role. In `manual` mode, the corresponding policy JSON files are also saved to the current directory. `manual` mode enables you to review the details before running the `aws` commands manually.
-<5> Links the {cluster-manager} IAM role to your Red Hat organization account.
+<1> Specify the prefix to include in the OCM IAM role name. The default is `ManagedOpenShift`. You can create only one OCM role per AWS account for your Red Hat organization.
+<2> Enable the admin {cluster-manager} IAM role, which is equivalent to specifying the `--admin` argument. The admin role is required if you want to use *Auto* mode to automatically provision the cluster-specific Operator roles and the OIDC provider by using {cluster-manager}.
```

Comment:
I know it was like this before, but should **Auto** be `auto` to match how you refer to it elsewhere in this module? I see the UI modules use **Auto**, probably since that's how it appears in the UI, so those instances make sense to me

Reply:
Yes in some cases it refers to the `auto` code references in the CLI and in this case it refers to the *Auto* UI object.